docs/api/browser-window.md

```diff
@@ -835,7 +849,7 @@ Moves window to `x` and `y`.
 
 #### `win.getPosition()`
 
-Returns an array that contains window's current position.
+Returns `Integer[]` - Contains the window's current position.
```

Comment:
So this syntax denotes "an array of Integers"? I think I remember seeing something more like `[Integer]` somewhere in the docs, but can't find any examples at the moment. Not saying that is a better syntax, but just want to be sure we're consistent.